lib/Doctrine/DBAL/Tools/Console/ConnectionNotFoundException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Doctrine\DBAL\Tools\Console;
+
+use OutOfBoundsException;
+
+class ConnectionNotFoundException extends OutOfBoundsException
```

Comment:
We don't use the `Exception` suffix for exceptions. `ConnectionNotFound` is descriptive enough. Please also mark `final`.